src/app/app.css

```diff
@@ -245,6 +245,42 @@ html, body, #root {
   overflow: auto;
 }
 
+.dashboard-card-resizable-wrapper { 
+  display: flex;
+  justify-content: space-between 
+}
+
+.pf-c-card.pf-m-rounded.dashboard-card {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.resizable-ref {
+  cursor: col-resize;
+  position: relative;
+  border-right: var(--pf-global--BorderWidth--lg) solid var(--pf-global--palette--black-500); /* 3px */
+  border-top-right-radius: var(--pf-global--BorderRadius--sm); /* 3px */
+  border-bottom-right-radius: var(--pf-global--BorderRadius--sm);
+}
+
+.draggable-ref__grip {
```

Comment:
I got an idea for this: Instead of adding a bottom line for header, we can make the card "float" similar so selectable cards when the header is hovered or dragged.

This can be achieved with a custom css class and set/reset this class on the DraggableRef.

```css
.draggable-ref__hover{
  box-shadow: 0 0.5rem 1rem 0 rgba(3, 3, 3, 0.16);
}
```

Notice AA card is having a shadow when hovering the header:

![float-card](https://user-images.githubusercontent.com/68053619/214902964-b6067f88-7b4c-4b9f-a039-18f7996f0078.png)